pkg/command/cert.go

```diff
@@ -59,8 +59,13 @@ func newCertCmd() *cobra.Command {
 		return cobra.AppendActiveHelp(nil, "Please provide 'host' or 'host:port'"), cobra.ShellCompDirectiveNoFileComp
 	}))
 
-	// The completion for this flag is simple file completion, which is configured by default
+	// --ca-certificate is renamed to --ca-cert
```

Comment:
From what I understand, this rename is to align with the terminology of the command which is `cert` and not `certificate`.  So this change is great.

What do you think however of using `--ca-cert-path`?
I find it would be clearer of what is expected as a flag value.
I compared with `imgpkg` and its similar flag is `--registry-ca-cert-path`; I don't think we need the `registry` prefix, but the `path` suffix may be useful.

Reply:
> What do you think however of using `--ca-cert-path`? I find it would be clearer of what is expected as a flag value. I compared with `imgpkg` and its similar flag is `--registry-ca-cert-path`; I don't think we need the `registry` prefix, but the `path` suffix may be useful.

I agree `-ca-cert-path` would be clear and helpful. However we have followed similar approach for `--*ca-certificate` for other command options for example [here](https://github.com/vmware-tanzu/tanzu-cli/blob/main/pkg/command/context.go#L210).

Reply:
Good point!  We need to consistent without our own tool.
Thanks for pointing this out to me.
Ok, so `--ca-cert` it is.